test/plugin/constants.rb

```diff
@@ -13,6 +13,33 @@
 # limitations under the License.
 
 # Constants used by unit tests for Google Cloud Logging plugin.
+module Google
+  module Protobuf
+    Any.class_eval do
+      def self.pack(msg, type_url_prefix = 'type.googleapis.com/')
+        any = Google::Protobuf::Any.new
+        any.pack(msg, type_url_prefix)
```

Comment:
This was google/protobuf#4719, which just got merged.

Reply:
Can we add a TODO to clean this up once we upgrade to the newer protobuf version?

Reply:
Friendly ping.